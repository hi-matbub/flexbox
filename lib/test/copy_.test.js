import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Copy from '../comps/copy_.jsx';

it('renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Copy />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders snapshot as expected', () => {
  const tree = renderer.create(<Copy />).toJSON();
  expect(tree).toMatchSnapshot();
});
