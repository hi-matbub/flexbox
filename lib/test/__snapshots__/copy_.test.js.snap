// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot as expected 1`] = `
<div
  className="codeblock empty"
>
  Try clicking above!
</div>
`;
